### DISCUSSION TITLE
Confirmed on AOOSTAR R7 5825U - Windows 11

### DISCUSSION THREAD

**Discussion by shinoki77** [isc30/ryzen-gpu-passthrough-proxmox#59] [General]
I was able to get iGPU passthrough working on the newer 5825U variant of the R7.  I also passed through my NVME controller.
I didn't have much luck with Bazzite so ended up sticking with Windows 11 and it's been working well so far.  

Couple notes:
- Had no luck getting the BIOS from the AliExpress seller or the Aoostar site but luckily the AMDGopDriver_5800H.rom on the main page worked to get rid of error 43.
- The RadeonResetBugFix worked initially but after a few reboots, I could not boot back into Windows.  It would just stay on a blackscreen, even after rebooting the Proxmox machine.  If i removed the gpu from the VM config Windows would boot normally.  I ended up doing another clean install of Windows without RadeonResetBugFix and have been able to turn off the VM on and off without issue.

### RESULTS
![Screenshot 2024-08-15 130750](https://github.com/user-attachments/assets/5510037d-260e-4684-8794-5286d48009ee)
![Screenshot 2024-08-15 130831](https://github.com/user-attachments/assets/f4c2b6ea-1d35-482a-b898-857708199ad2)
<img width="1170" alt="Screenshot 2024-08-15 at 13 16 44" src="https://github.com/user-attachments/assets/fad08477-5707-46b5-b61c-f0d68347856c">

Attached is the [vbios_5825U BIOS](https://github.com/user-attachments/files/16628444/vbios_5825U.bin.zip)

**Comment by dutchmhk** [comment]
I have a WTR PRO with same CPU and presumably same Bios. I have a hard time passing through the IGPU to any VM (it's needed for Frigate NVR) and I end up breaking Proxmox trying various settings. Been following several guides but to no avail. Would you be so kind and point me in the right direction and maybe share your Bios settings. Any Help is much appreciated. Cheers.

**Comment by Primzy6551** [comment]
I followed this guide but couldn't make it work. Did you do anything special? Cheers

**Comment by shinoki77** [comment]
Which part are you stuck at?

**Reply by Primzy6551** [reply]
I somehow managed it to make it work. But it has sub par performance that i was expecting. Some graphical glitches.
